AI4PEX_Data_Questionnaire/questionnaire.md

```diff
@@ -0,0 +1,61 @@
+We'd appreciate it if you had 5min to take this survey, this will help us gauge the amount of compute
+resources we'd need to have for the AI4PEX project (both available, and to request new allocations). Very many thanks 
+in advance! :beer:
+
+- 1. What datasets are you planning to use in your work in EXPECT? Choose as many as you like:
+  - CMIP6
+  - LSFMIP
+  - Climate Digital Twin DestinE
+  - EERIE, NextGEMS
+  - Mars seasonal forecasts
+  - ERA5
+  - ESA CCI observations
+  - Simulations generated within AI4PEX
+  - Not sure (yet)
+  - Others (please tell us)
```

Comment:
NEMO is the ocean model so perhaps you can add that as an option?

Reply:
good call! Done in https://github.com/AI4PEX/Housekeeping/pull/3/commits/f638a79ada95d372f5e622f81b1c8ab8db8b883a